lib/ssh_scan/update.rb

```diff
@@ -5,6 +5,10 @@
 
 module SSHScan
   class Update
+    def initialize
+      @errors = []
+    end
```

Comment:
If we're adding this, we should probably make use of it.  Otherwise, maybe it can wait until we want to do use it.